HACKING.md

```diff
@@ -0,0 +1,29 @@
+
+#Hacking On The Manifesto
+
+###Background
+
+The manifesto acts as a sort of code of conduct for developers down with the Namespace App Club movement. While we won't
```

Comment:
It's Namespace but again the name is vague so I put App Club in so people know it's an App Club. For example someone who came here through Google.